cluster-scope/base/rbac.authorization.k8s.io/clusterroles/kuberay-operator/clusterrole.yaml

```diff
@@ -0,0 +1,214 @@
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  creationTimestamp: null
```

Comment:
Please remove null timestamp, this can lead to sync issues in ArgoCD.

Reply:
done